source/reference/operator/query-miscellaneous.txt

```diff
@@ -0,0 +1,37 @@
+=============================
+Miscellaneous Query Operators
+=============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/extracts/operators-toc-explanation.rst
+
+.. list-table::
+   :widths: 30,70
+   :header-rows: 1
```

Comment:
Can you bump the column width slightly so that the description field doesn't wrap by one character please?

Reply:
See above